src/inject/dynamic-theme/index.ts

```diff
@@ -141,9 +141,15 @@ function createShadowStaticStyleOverrides(root: ShadowRoot) {
     const inlineStyle = createOrUpdateStyle('darkreader--inline', root);
     inlineStyle.textContent = getInlineOverrideStyle();
     root.insertBefore(inlineStyle, root.firstChild);
+
+    const userAgentStyle = createOrUpdateStyle('darkreader--user-agent');
+    userAgentStyle.textContent = getModifiedUserAgentStyle(filter, isIFrame, filter.styleSystemControls);
```

Comment:
Maybe let's cache the text content of user agent style some way? Some pages can have many custom elements.

Reply:
it could be found in another book in page #1772569

hope you can find your answer

`
```suggestion
    userAgentStyle.textContent = getModifiedUserAgentStyle(filter, isIFrame, filter.styleSystemControls);
    
```